roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -97,12 +97,39 @@ def get_var(self, *keys, **kwargs):
         OpenShiftCheckException when a key is not found or returning a default value
         provided as a keyword argument.
         """
+        if len(keys) == 1:
+            keys = keys[0].split(".")
+
         try:
             value = reduce(operator.getitem, keys, self.task_vars)
         except (KeyError, TypeError):
             if "default" in kwargs:
                 return kwargs["default"]
             raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
+
+        convert = kwargs.get("type")
```

Comment:
although, note the logic under bool... part of the reason for doing this is to not have to repeat that everywhere.

I suppose we could still special-case known conversion functions like bool though.